executor/joiner.go

```diff
@@ -460,3 +460,13 @@ func (j *innerJoiner) tryToMatch(outer chunk.Row, inners chunk.Iterator, chk *ch
 
 func (j *innerJoiner) onMissMatch(_ bool, outer chunk.Row, chk *chunk.Chunk) {
 }
+
+// IsOuterJoiner returns if this joiner is a outer joiner
+func IsOuterJoiner(j plannercore.JoinType) bool {
```

Comment:
how about bind this function to `JoinType`? thus we can call `joinType.IsOuterJoin()` to achieve this functionality.

Reply:
updated.